README.md

```diff
@@ -93,6 +93,148 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
+```
+ 
+
+Example 2: Ignore a specific set of security vulnerabilities present in the flask package in a pypi manifest file.
+```
+flask==1.0 #crdaignore [vulnerability1, vulnerability2, ..., vulnerability n]
+fastapi==0.36.0 
+sceptre==2.2.1 
+```
+
+Ignore all the security vulnerabilities present in the "ginkgo" and "pax-go" dependencies in a golang manifest file.
+```
+module github.com/fabric8-analytics/acceptance_tests/data
+
+go 1.14
+
+require (
+	code.cloudfoundry.org/archiver v0.0.0-20170223024658-7291196139d7
+	github.com/googleapis/gax-go v1.0.3 crdaignore
+	github.com/googleapis/gax-go/v2 v2.0.5
+	github.com/onsi/ginkgo v1.14.2 // indirect crdaignore
+	github.com/onsi/gomega v1.10.3 // indirect 
+)
+
+```
+Ignore all vulnerabilities of "pax-go", and ignore a specific set of security vulnerabilities present in the "gingko" dependency in a golang manifest file.
+```
+module github.com/fabric8-analytics/acceptance_tests/data
+
+go 1.14
+
+require (
+	code.cloudfoundry.org/archiver v0.0.0-20170223024658-7291196139d7
+	github.com/googleapis/gax-go v1.0.3 crdaignore
+	github.com/googleapis/gax-go/v2 v2.0.5
+	github.com/onsi/ginkgo v1.14.2 // indirect crdaignore [vulnerability 1, vulnerability 2]
+	github.com/onsi/gomega v1.10.3 // indirect 
+)
+```
+Ignore all vulnerabilities of the dependency "junit:junit". 
+```
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.bayessian.test</groupId>
+  <artifactId>test-app-junit-dependency</artifactId>
+  <version>1.0</version>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>  <!--crdaignore-->
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+    </dependency>
+  </dependencies>
+</project>
+```
+Note: To ignore vulnerabilities for a dependency in a maven manifest file, insert "crdaignore" in comments against the group id, artifact id, or version of that particular dependency.
+
+Ignore "vulnerability 1" and "vulnerability 2" of the dependency "junit:junit". 
+```
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.bayessian.test</groupId>
+  <artifactId>test-app-junit-dependency</artifactId>
+  <version>1.0</version>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>  
+      <artifactId>junit</artifactId>   <!--crdaignore [vulnerability 1, vulnerability 2]-->
+      <version>3.8.1</version>
+    </dependency>
+  </dependencies>
+</project>
+```
+Ignore vulnerabilities of packages in an npm manifest file - To Ignore vulnerabilities of npm packages, insert the "crdaignore" JSON object. The "crdaignore" object consists of the object "packages", which consists of the packages to ignore as keys, and the list of vulnerabilities to ignore as values. To ignore all the security vulnerabilities of an npm package, insert "*" in the list of vulnerabilities to ignore as shown in the example below. The below example ignores all the vulnerabilities of "bootstrap" and "vulnerability 1" of the lodash package.
```

Comment:
Explained it above so description is not needed, just give proper example to explain the format.